SpectodaSound.js

```
@@ -112,33 +113,34 @@ export class SpectodaSound {
-            logging.debug("SpectodaSound.connect", "Connected SystemSound");
+            logging.debug('SpectodaSound.connect', 'Connected SystemSound');
           })
-          .catch(e => {
+          .catch((e) => {
             reject(e);
-            throw "DeviceUnsupported";
+            throw 'DeviceUnsupported';
           });
       });
     } else {
       this.#stream = mediaStream;
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)